bundles/org.dataflowanalysis.analysis.dfd/src/org/dataflowanalysis/analysis/dfd/core/DFDCyclicTransposeFlowGraphFinder.java

```diff
@@ -113,7 +113,7 @@ private List<DFDVertex> determineSinks(DFDVertex sink, List<Pin> inputPins, List
             List<DFDVertex> finalVertices = vertices;
             
             vertices = incomingFlowsToPin.stream()
-                    .flatMap(flow -> handleIncomingFlow(flow, inputPin, finalVertices, sourceNodes, previousNodesInTransposeFlow).stream())
+                    .flatMap(flow -> handleIncomingFlowCyclic(flow, inputPin, finalVertices, sourceNodes, previousNodesInTransposeFlow).stream())
```

Comment:
is changing the name necessary?
If you don´t need the cyclic finder part but only the standard finder in your build of the analysis you can 
provide the class of the wanted flow gtaph finder

Reply:
I changed the handleIncomingFlow method in the normal finder by adding a list parameter. With the cycling finder inheriting from the normal one, this leads to a similar signature since List is List for Java, no matter what type